### PR TITLE
Add Cloud SOAR Audit Index documentation

### PR DESCRIPTION
**Pull request by pbaroni**
## Purpose of this pull request

Add Audit Features-_sourceCategory mapping available in CSOAR Delivery 2.
The table is incorporated in a new page in Cloud SOAR section.
Minor titles moving and formatting issues.

Issue number: CSOAR-907

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [X] Update Content - Revisions and updating sections
- [X] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/cloud-soar/incidents-triage.md [hunk 1]
```diff
@@ -116,7 +116,7 @@ The Incident Details section contains all the important information that makes u
 
 ### Overview
 
-The Incident Overview section contains all the pertinent information for a specific incident such as the severity, SLA counter, and category of alert. This information can be customized in the Custom Fields section of the platform. For more information, please see [**Custom Fields**](#Custom-Fields-1).
+The Incident Overview section contains all the pertinent information for a specific incident such as the severity, SLA counter, and category of alert. This information can be customized in the Custom Fields section of the platform. For more information, please see [**Custom Fields**](#Custom-Fields).
```
Comment (jpipkin1) [inline docs/cloud-soar/incidents-triage.md:119]:
```suggestion
The Incident Overview section contains all the pertinent information for a specific incident such as the severity, SLA counter, and category of alert. This information can be customized in the Custom Fields section of the platform. For more information, please see [Custom Fields](#custom-fields).
```

**Comment by jpipkin1** [comment]
@pbaroni, per our conversation, I made this PR into a draft. When you are done placing the audit event index table where it should go, then click "Read for review" to take it out of draft and put it into review.

**Comment by pbaroni** [comment]
@jpipkin1 The feature has been released in Prod deployments: this PR can be reviewed and merged in public docs.
Thank you.

File: docs/cloud-soar/incidents-triage.md [hunk 2]
```diff
@@ -116,7 +116,7 @@ The Incident Details section contains all the important information that makes u
 
 ### Overview
 
-The Incident Overview section contains all the pertinent information for a specific incident such as the severity, SLA counter, and category of alert. This information can be customized in the Custom Fields section of the platform. For more information, please see [**Custom Fields**](#Custom-Fields-1).
+The Incident Overview section contains all the pertinent information for a specific incident such as the severity, SLA counter, and category of alert. This information can be customized in the Custom Fields section of the platform. For more information, please see [Custom Fields](#custom-fields).
```
Comment (jpipkin1) [inline docs/cloud-soar/incidents-triage.md:119]:
```suggestion
The Incident Overview section contains all the pertinent information for a specific incident such as the severity, SLA counter, and category of alert. This information can be customized in the Custom Fields section of the platform. For more information, see [Custom Fields](#custom-fields).
```

File: docs/cloud-soar/incidents-triage.md [hunk 3]
```diff
@@ -371,26 +373,26 @@ Fields may be reorder in the Custom Fields section to change the order in which
...
-URL | URL Available to use in actions which require a URL input
-User Details | User details, such as a user name. Available to use in actions which require a user details input
+| Field Type        | Description                                                                                       |
+|:------------------|:--------------------------------------------------------------------------------------------------|
+| Calculation  | Perform a calculation between two fields or between a field and a static value. |
+| Checkbox   | Checkbox. |
+| Color Picker  | Interactive color picker to select a color. |
+| Date | Date only picker. |
+| Date & Time   | Date and time picker. |
+| Email Address | Email address Available to use in actions which require a email input.    |
```
Comment (jpipkin1) [inline docs/cloud-soar/incidents-triage.md:383]:
```suggestion
| Email Address | Email address available to use in actions which require a email input.    |
```

File: docs/cloud-soar/index.md [hunk 4]
```diff
@@ -69,4 +69,10 @@ Navigate the evolving threat landscape with our best-in-class security orchestra
   <p>Learn how to access Cloud SOAR APIs and API documentation.</p>
   </div>
 </div>
+<div className="box smallbox10 card">
```
Comment (jpipkin1) [inline docs/cloud-soar/index.md:72]:
```suggestion
<div className="box smallbox card">
```

**Review by jpipkin1 — APPROVED** [review]
You did a great job, @pbaroni! I made some small updates, including moving the table from a separate file into the main article (see commit [Add table and make minor edits](https://github.com/SumoLogic/sumologic-documentation/pull/3183/commits/9954081ebe70790151dbeeb193f39fdd5fe15efb)). There is no need to use an import to bring in a separate file unless that file is being [resued](https://help.sumologic.com/docs/contributing/style-guide/#reusing-content).

**Pull request merged**